### PR TITLE
Initialise the Game

### PR DESCRIPTION
**Pull request by jennysivapalan**
* Game service which will initalise the number of players, number of rounds and ensures other variables have correct initial values.
* Uses this on App.tsx
* It uses dummy data for the cards

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
 import { Header } from "./components/header/Header";
+import { initialiseGame } from "./service/game";
 function App() {
+  const game = initialiseGame(DEFAULT_PLAYERS, DEFAULT_ROUNDS);
+  const player1 = game.players[0];
+  const player2 = game.players[1];
+
   return (
     <>
       <PlayerScore
```
Comment (jennysivapalan) [inline client/src/App.tsx:13]: @beccarothwell I was thinking we could pass to the PlayerScore the player as an object and then get the name, id, score in the component? What do you think?

Comment (beccarothwell) [reply client/src/App.tsx:13]: I considered this at the time, but it was before we defined types. That makes sense though, I'll do a PR for that later.

Comment (beccarothwell) [reply client/src/App.tsx:13]: Looks good, but the game object needs to be held in State, then we can derive players, currentRound etc from that state.

Comment (jennysivapalan) [reply client/src/App.tsx:13]: Yep - do we need to do that now though or when we get to it?

Comment (jennysivapalan) [reply client/src/App.tsx:17]: added

**Review by beccarothwell — APPROVED** [review]
Great thanks

**Pull request merged**